script.js

```diff
@@ -1,4 +1,221 @@
+var win = 0;
+var loss = 0;
+var currentMode = "Waiting for username";
+var username = "";
+var lastroundWinner = "";
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
```

Comment:
good job keeping your logic succinct! However, your main function is a bit too long. Generally, a function should not be over 3/4 of your screen. If you see your function stretches past a full screen, it should signal to you that maybe you can try to split it up into helper functions. This is to improve the readability of your code as the reader would not have to look through each line of code to know what the function does. So in this case, it would be good if you could section out the logic for each game mode into individual helper functions to be called in your main function